common/src/main/java/com/wynntils/commands/ConfigCommand.java

```diff
@@ -2,14 +2,18 @@
  * Copyright © Wynntils 2022.
  * This file is released under AGPLv3. See LICENSE for full license details.
  */
-package com.wynntils.commands.wynntils;
+package com.wynntils.commands;
+
+import static net.minecraft.commands.Commands.literal;
```

Comment:
Ew. Do we like static imports?

Reply:
In general, no, but I thought it made the complex Brigadier node building much more readable. But if you really dislike it, I can remove them.

Reply:
Isn't it nice to remove it to keep to our standards?

Reply:
OK, I'll fix.